promscale/visualize-data/jaeger.md

```diff
@@ -0,0 +1,38 @@
+# Connect Promscale and Jaeger Query
```

Comment:
The title for this page should be more clear about the fact this is about Visualizing data and not other things.

Maybe "Visualize traces in Jaeger" or "Visualize Promscale traces in Jaeger" (which may be better for SEO if people search Promscale and Jaeger).